centos-ci/clang-analysis/run-clang-analysis.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+# if anything fails, we'll abort
+set -e
+
+function install_dependencies()
+{
+    sudo yum -y install automake autoconf libtool flex bison openssl-devel \
+                libxml2-devel python-devel libaio-devel libibverbs-devel \
+                librdmacm-devel readline-devel lvm2-devel glib2-devel \
+                userspace-rcu-devel libcmocka-devel libacl-devel make \
+                gcc gdb git hostname libattr-devel yajl-devel clang-analyzer
+}
+
+function update_gerrit_status()
+{
+    if [ ${RET} = 0 ]; then
+        local V="+1"
+        local VERDICT="SUCCESS"
+    else
+        local V="-1"
+        local VERDICT="FAILED"
+    fi
+
+    local BURL=${BUILD_URL}consoleFull
+
+    # Update Gerrit with the success/failure status
+    ssh build@review.gluster.org gerrit review \
```

Comment:
This needs to be part of the Jenkins job (xml). The machine that runs the test does not have any ssh credentials to post something to Gerrit.